crates/ruff/resources/test/fixtures/flake8_comprehensions/C414.py

```diff
@@ -7,11 +7,14 @@
 set(list(x))
 set(tuple(x))
 set(sorted(x))
+set(sorted(x, key=lambda y: y))
 set(reversed(x))
 sorted(list(x))
 sorted(tuple(x))
 sorted(sorted(x))
+sorted(sorted(x, key=lambda y: y))
```

Comment:
If I'm reading the description and code right, this is no longer flagged. But isn't this the same as just `sorted(x)`? Doesn't the inner sorted get "undone" by the outer sorted? Same with `set(sorted(x, key=lambda y: y))`.

Reply:
Oh right, I completely missed that. I was thinking from a different perspective. This means that the inner arguments need not be preserved and only the outer ones should be preserved.

Reply:
Ok, so the solution is to not preserve the inner arguments except for the `iterable`.